spec.html

```diff
@@ -28440,6 +28440,209 @@ <h1>
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-create-default-export-synthetic-module" type="abstract operation">
+          <h1>
+            CreateDefaultExportSyntheticModule (
+              _defaultExport_: an ECMAScript Language value,
+            ): a Synthetic Module Record
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It creates a Synthetic Module Record whose default export is _defaultExport_.</dd>
+          </dl>
+          <emu-alg>
+            1. Let _realm_ be the current Realm Record.
+            1. Let _setDefaultExport_ be a new Abstract Closure with parameters (_module_) that captures _defaultExport_ and performs the following steps when called:
+              1. Perform SetSyntheticModuleExport(_module_, *"default"*, _defaultExport_).
+            1. Return CreateSyntheticModule(« *"default"* », _setDefaultExport_, _realm_).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-parse-json-module" type="abstract operation">
+          <h1>
+            ParseJSONModule (
+              _source_: a String,
+            ): either a normal completion containing a Synthetic Module Record, or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd></dd>
+          </dl>
+
+          <emu-alg>
+            1. Let _json_ be ? Call(%JSON.parse%, *undefined*, « _source_ »).
+            1. Return CreateDefaultExportSyntheticModule(_json_).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-setsyntheticmoduleexport" type="abstract operation">
+          <h1>
+            SetSyntheticModuleExport (
+              _module_: a Synthetic Module Record,
+              _exportName_: a String,
+              _exportValue_: an ECMAScript language value,
+            ): ~unused~
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It can be used to set or change the exported value for a pre-established export of a Synthetic Module Record.</dd>
+          </dl>
+
+          <emu-alg>
+            1. Assert: _module_.[[ExportNames]] contains _exportName_.
+            1. Let _envRec_ be _module_.[[Environment]].
+            1. Assert: _envRec_ is not ~empty~.
+            1. Perform _envRec_.SetMutableBinding(_exportName_, _exportValue_, *true*).
+            1. Return ~unused~.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-smr-module-record-methods">
+          <h1>Implementation of Module Record abstract methods</h1>
+
+          <p>The following are the concrete methods for Synthetic Module Record that implement the corresponding Module Record abstract methods defined in <emu-xref href="#table-abstract-methods-of-module-records"></emu-xref>.</p>
+
+          <emu-clause id="sec-smr-LoadRequestedModules" type="concrete method">
```

Comment:
There is an open PR in the proposal changing these `smr` shorthands to the longer `sec-synthetic-module-record-LoadRequestedModules`. What is the editorial preference?

Reply:
I am completely unopinionated on IDs. This ID is fine.